lib/Serverless.js

```diff
@@ -81,6 +81,7 @@ class Serverless {
     if (this.cli.displayHelp(this.processedInput)) {
       return BbPromise.resolve();
     }
+    this.cli.suppressLog(this.processedInput);
```

Comment:
could you just add a comment here that explains that logs would only be suppressed if the print command is entered? Just to make it clear that logs don't always get suppressed, since all commands will run this method.

or maybe call the method `suppressLogIfPrintCommand`? 😄

Reply:
Renamed in https://github.com/serverless/serverless/pull/5728/commits/59c440e0737801eb37de63dc74e133be8bbb83f7